controllers/utils/constants.go

```diff
@@ -137,3 +137,7 @@ const (
 	PlcHubTmplPrinfInNameErr = "printf variable is not supported in the template function Name field"
 	PlcHubTmplPrinfInNsErr   = "printf variable is not supported in the template function Namespace field"
 )
+
+// SpaceRequiredForPrecache is an env variable for precaching job that indicates the amount of space required
+// for precahing job. This is an rough estimate. 10 GB for OCP images and 5 GB for operator images.
```

Comment:
```suggestion
// for precaching job. This is a rough estimate. 10 GB for OCP images and 5 GB as a buffer for operator images and other disk allocations that can happen between the job and the actual upgrade kicked off by the CGU.
```

Reply:
This is not incorporated?

Reply:
I think it got lost when I merged the master into it :))